internal/pool/pool_test.go

```diff
@@ -19,8 +19,6 @@ import (
 )
 
 type testItem struct {
-	v uint32
```

Comment:
This line was removed because the structcheck linter wrote `internal/pool/pool_test.go:22:2: `v` is unused (structcheck)
        v uint32
`